src/porepy/models/abstract_equations.py

```diff
@@ -183,3 +184,57 @@ def perturbation_from_reference(self, variable_name: str, grids: list[pp.Grid]):
         d_var = var(grids) - var_ref(grids)
         d_var.set_name(variable_name + "_perturbation")
         return d_var
+
+
+class BoundaryConditionsMixin:
```

Comment:
I think I'd prefer singular, i.e. BoundaryCondition. And does this perhaps belong in a separate file?

Reply:
BoundaryCondition or BoundaryConditionMixin? I agree on moving to a separate file.

Reply:
I would prefer BoundaryCondition, but that's already taken in bc.py (as is AbstractBoundaryCondition). I can't think of a better name than BoundaryConditionMixin. Feel free to suggest!

Reply:
I like BoundaryConditionMixin